.travis.yml

```diff
@@ -5,6 +5,3 @@ script:
 - npm test --coverage
 #- npm run-script coveralls
 - npm run-script lint
-env:
```

Comment:
Will need to get #ops or whoever has travis access to set this up? Or can anyone use the public key to encrypt this? In any case, travis will be need to be enabled for ripple/ripple-keypairs